pkg/apis/core/types_shoot.go

```diff
@@ -516,6 +519,13 @@ type ClusterAutoscaler struct {
 	Verbosity *int32
 }
 
+type ExpanderConfig struct {
+	// ConfigMapName specifies the name of the expander configuration, according to the priority expander.
+	// See: https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/expander/priority/readme.md
+	// +optional
+	ConfigMapName *string `json:"configMapName,omitempty"`
```

Comment:
Remove this from internal APIs.
```suggestion
	ConfigMapName *string
```